_changelog-files/2021/2021-06-28-facebook-ads-v1-update-country-primary-key.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Facebook Ads (v1) update: Fixed composite primary key for the `ads_insights_country` table"
+content-type: "changelog-entry"
+date: 2021-06-28
+entry-type: bug-fix
+entry-category: "integration"
+connection-id: "facebook-ads"
+connection-version: "1"
+pull-request: "https://github.com/singer-io/tap-facebook/pull/154"
+---
+{{ site.data.changelog.metadata.single-integration | flatify }}
+
+We've fixed the `ads_insights_country` composite primary key. The `country` field had previously been erroneously excluded.
+
+The composite primary key is now: `campaign_id", "adset_id", "ad_id", "date_start", "country"`.
+
+This is a breaking change and communiation from Stitch support has been emailed to affected users.
+
+If you are not already including the `country` field in downstream processes that identify uniqueness of records in the files loaded to S3 for this table, this field will need to be added into those processes.
```

Comment:
Why the mention of S3? This isn't specific to S3 destinations, and our internal processes (which do involve an S3 bucket) aren't relevant here.